write-fonts/src/graph.rs

```diff
@@ -862,7 +864,7 @@ impl Graph {
     fn actually_promote_subtables(&mut self, to_promote: &[ObjectId]) {
         fn make_extension(type_: LookupType, subtable_id: ObjectId) -> TableData {
             const EXT_FORMAT: u16 = 1;
-            let mut data = TableData::new(type_.promote().into());
+            let mut data = TableData::new(TableType::Named("ExtensionPosFormat1"));
```

Comment:
The named by string scares me, we can't use some sort of type here?

Reply:
This is just used for diagnostics, and in every other case is set during codegen.